lib/App/DuckPAN/Cmd/New.pm

```diff
@@ -51,33 +103,90 @@ sub run {
 		$lc_name = $lc_path . "_" . $lc_name;
 	}
 
-	$self->app->emit_and_exit(-1, "No templates exist for this IA Type: " . $type->{name}) if (!defined $type->{templates});
-
-	my %template_info = %{$type->{templates}};
-	my $tx            = Text::Xslate->new();
-	my %files         = (
-		test       => ["$filepath.t"],
-		code       => ["$filepath.pm"],
-		handlebars => [$lc_filepath, "$lc_name.handlebars"],
-		js         => [$lc_filepath, "$lc_name.js"]);
 	my %vars = (
 		ia_name           => $name,
 		ia_package_name   => $package_name,
 		ia_name_separated => $separated_name,
 		lia_name          => $lc_name,
-		ia_path           => $filepath
+		ia_path           => $filepath,
+		lia_id            => $lc_name,
 	);
-	foreach my $template_type (sort keys %template_info) {
-		my ($source, $dest) = ($template_info{$template_type}{in}, $template_info{$template_type}{out});
-		$self->app->emit_and_exit(-1, 'Template does not exist: ' . $source) unless ($source->exists);
-		# Update dest based on type:
-		$dest = $dest->child(@{$files{$template_type}});
-		$self->app->emit_and_exit(-1, 'File already exists: "' . $dest->basename . '" in ' . $dest->parent) if ($dest->exists);
-		my $content = $tx->render("$source", \%vars);
-		$dest->touchpath->append_utf8($content);    #create file path and append to file
-		$self->app->emit_info("Created file: $dest");
+
+	# Ask which optional templates to create
+	my @optional_templates;
+
```

Comment:
Sounds good, thanks. I'll make this change. I have a clarification though - does the order of the options matter? For example, can `css` be option 1 and `javascript` be 2? And in 5 for example, can the options be `css, handlebars, javascript`?

Should we also expand the idea of showing the menu to listing the templates when the `--template` argument is not passed? Such as:
`
$ duckpan new
Which template would you like to use for your instant answer?

    1> standard (Standard Goodie instant answer with a Perl module and optional frontend files)
    2> cheat sheet (Cheat Sheet with the cheat sheet JSON file)

Choose configuration [1]:
`